packages/@dcl/inspector/src/components/EntityInspector/GltfInspector/GltfInspector.tsx

```diff
@@ -23,7 +22,7 @@ const DROP_TYPES = ['project-asset-gltf']
 
 export default withSdk<Props>(
   withContextMenu<WithSdkProps & Props>(({ sdk, entity, contextMenuId }) => {
-    const { files } = useFileSystem()
+    const [files] = useFileSystem()
```

Comment:
Don't get it this change, is there any difference?

Reply:
just for consistensy. We were returning arrays for hooks, and also this is the way useEffect uses.